elliottlib/bzutil.py

```diff
@@ -61,7 +61,7 @@ def is_tracker_bug(self):
         return set(constants.TRACKER_BUG_KEYWORDS).issubset(set(self.keywords))
 
     def is_cve_in_summary(self):
-        return True if re.search(r'^CVE-\d+-\d+', self.summary) else False
+        return bool(re.search(r'^CVE-\d+-\d+', self.summary))
```

Comment:
`^CVE-\d+-\d+` will only match strings that start with `CVE-`. I guess it could be better to remove `^` to let it match whatever `CVE-*-*` substring in arbitrary location.

Reply:
hmm not sure if that is required. If prodsec automation had something to do with creating the bug, the first word should start with `CVE`. Do we optimize for false negative or false positives?
I'd be erring on the side of allowing devs to use the word `CVE` somewhere in the summary (and have prodsec fix it up if the tooling did not create the tracker correctly).

Reply:
This PR is not perfect but it will improve things. So I think I am ok to merge it.